### PR TITLE
Added signing keys to operator claims.

### PR DESCRIPTION
**Pull request by sasbury**

### CODE REVIEW

**Comment by sasbury** [comment]
Let me add that.

File: operator_claims.go [hunk 1]
```diff
@@ -45,6 +45,31 @@ func NewOperatorClaims(subject string) *OperatorClaims {
...
+		return false
+	}
+
+	issuer := op.Claims().Issuer
+
+	if issuer == s.Subject {
+		return true
+	}
+
+	if s.SigningKeys == nil {
```
Comment (derekcollison) [inline operator_claims.go]: Don't need this test, range works over nil.
